### PR TITLE
fix: Position of arrows in Table chart

### PR DESCRIPTION
**Pull request by geido**
### SUMMARY
Fixes the position of the arrows in the Table chart

### BEFORE

Screen Shot 2022-02-15 at 13 27 57

### AFTER

Table chart AFTER

### TESTING INSTRUCTIONS
1. Open a Table chart
2. Add several columns
3. Make sure the cells are aligned correctly

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/plugins/plugin-chart-table/src/Styles.tsx [hunk 1]
```diff
@@ -40,8 +40,9 @@ export default styled.div`
   }
   th svg {
     color: ${({ theme: { colors } }) => colors.grayscale.light2};
-    position: relative;
+    position: absolute;
     vertical-align: middle;
+    top: ${({ theme: { gridUnit } }) => gridUnit * 2}px;
```
Comment (kgabryje) [inline superset-frontend/plugins/plugin-chart-table/src/Styles.tsx:45]: Should we add `right: 0`? You can see on the screenshot that the arrows overlap with the th text on the right
image

Comment (geido) [reply superset-frontend/plugins/plugin-chart-table/src/Styles.tsx:45]: Nice catch. However, I could not reproduce it locally. That's weird. I pushed the change with your suggestion.

**Review by kgabryje — APPROVED** [review]
Love the styles clean up! LGTM!

**Pull request merged**